assignments/lambda-classes.js

```diff
@@ -1 +1,72 @@
 // CODE here for your Lambda Classes
+
+class Person {
+
+    constructor(attributes){
+
+      this.name = PersonAttributes.name;
+      this.age = PersonAttributes.age;
+      this.location = PersonAttributes.location;
+      this.gender = PersonAttributes.gender;
+
+    }
+    
+    speak(){
+      return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+}
+
+class Instructors extends Person {
+    constructor (instructorAttributes){
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+
+    demo('subject'){
+        return `Today we are learning about ${subject}`
+    }
+
+    grade('subject'){
+        return `${student.name} receives a perfect score on ${subject}`
+    }
+
+}
+
+class Student extends Instructors {
+    constructor (studentAttributes){
+        super(studentAttributes);
+        this.previousBackground = studentAttributes.previousBackground;
+        this.className = studentAttributes.className;
+        this.favSubjects = studentAttributes.favSubjects;
+    }
+
+    listsSubjects(){
+        return `${this.favSubjects}`
```

Comment:
I believe this method is supposed to list out each subject one by one. You could do this with the use of a loop